slacktee.sh

```diff
@@ -22,8 +22,8 @@
 # ----------
 # Default Configuration
 # ----------
-webhook_url=""       # Incoming Webhooks integration URL
-token=""             # The user's API authentication token, only used for file uploads or streaming
+webhook_url=""       # Incoming Webhooks integration URL. Old way to interact with Slack. (Deprecated)
```

Comment:
Could we log a deprecation message to stderr if `webhook_url` is set?

Reply:
Ah! Good idea! Let me implement it :)

Reply:
Implemented!